src/Paybox.php

```diff
@@ -48,15 +49,49 @@ public function __construct(array $options = [], AbstractHttpClient $httpClient
     /**
      * @param RequestInterface $request
      *
-     * @return PayboxResponse
+     * @return DirectResponse
+     */
+    public function requestDirect(RequestInterface $request)
```

Comment:
This method name does not feel right, it looks like you're "requesting a direct", which is not very meaningful… maybe `sendDirectRequest` and `sendDirectPlusRequest`